### PR TITLE
Migrate from use npm to yarn

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+	@if ! [ $(shell which yarn) ]; then \
+		echo "yarn is not installed"; \
 		exit 1; \
 	fi
 
-	@echo Getting dependencies using npm
+	@echo Getting dependencies using yarn
 
-	npm install --ignore-scripts --no-package-lock
+	yarn install --ignore-scripts --no-package-lock
```
Comment (enahum) [inline Makefile]: does yarn has a `--ignore-scripts` flag?

do we really need to set `--no-package-lock` ?

Comment (jespino) [reply Makefile]: Yes, and I don't know.

Comment (enahum) [reply Makefile]: can you please test it and then report back?

Comment (jespino) [reply Makefile]: Yes sorry, I tested look over the help too fast and my mind made the match --no-lockfile with --no-package-lock automatically :(

`--pure-lockfile` sounds good to me

**Pull request merged**